### PR TITLE
Initial setting

### PR DESCRIPTION
**Pull request by OlhaKolos**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Sensitive or high-churn files
+.idea/**/dataSources/
```
Comment (NovoDan) [inline .gitignore]: add whole .idea folder instead separate inner folders

File: src/test/java/TestExample.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+@Test()
```
Comment (NovoDan) [inline src/test/java/TestExample.java]: brackets are redundant here

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Sensitive or high-churn files
```
Comment (ihor-dynka) [inline .gitignore]: you can just specify entire folder .idea/

File: src/main/java/pom/components/AbstractFragment.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package pom.components;
+
```
Comment (ihor-dynka) [inline src/main/java/pom/components/AbstractFragment.java:2]: if the name contains Abstract then make the class abstract

**Review by ihor-dynka — CHANGES_REQUESTED** [review]

**Review by NovoDan — APPROVED** [review]

**Review by ihor-dynka — APPROVED** [review]

**Pull request merged**